01_pipeline/01_apply_manifest_task.yaml

```diff
@@ -1,16 +1,16 @@
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
```

Comment:
could you make it `triggers.tekton.dev`

Reply:
and it all the other Triggers Resources

Reply:
please ignore this :D

Reply:
I think `triggers.tekton.dev` only for trigger resources not for pipeline resources right